api/zip_test.go

```diff
@@ -0,0 +1,36 @@
+package api
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/qri-io/dataset"
+)
+
+func TestGetZip(t *testing.T) {
```

Comment:
Would prefer to see all of this moved into `get_test.go`

Reply:
Do you mean rename this to `get_test.go`? There isn't a file with that name in the `api/` package.

Reply:
Ooooops, yes rename then!